ebos/ecloutputblackoilmodule.hh

```diff
@@ -662,6 +662,11 @@ private:
             if (FluidSystem::phaseIsActive(oilPhaseIdx)) {
                 this->pressureTimesPoreVolume_[globalDofIdx] = getValue(fs.pressure(oilPhaseIdx)) * pv;
                 this->pressureTimesHydrocarbonVolume_[globalDofIdx] = this->pressureTimesPoreVolume_[globalDofIdx] * hydrocarbon;
+            } else if (!FluidSystem::phaseIsActive(oilPhaseIdx) && (FluidSystem::phaseIsActive(gasPhaseIdx))) {
```

Comment:
Since the first condition here is `phaseIsActive(oil)` you can simplify this condition to
```C++
else if (FluidSystem::phaseIsActive(gasPhaseIdx))
```